src/content/en/2022/webassembly.md

```diff
@@ -1,22 +1,160 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: WebAssembly
-#TODO - Review and update chapter description
-description: WebAssembly chapter of the 2022 Web Almanac covering Wasm compression, section sizes, most popular instructions, and post-MVP proposals
+description: WebAssembly chapter of the 2022 Web Almanac covering usage, languages, and post-MVP features.
 authors: [ColinEberhardt]
 reviewers: [binji, RReverser]
 analysts: [JamieWhitMac]
-editors: []
+editors: [tunetheweb]
 translators: []
+ColinEberhardt_bio: Colin is Technology Director at <a hreflang="en" href="https://www.scottlogic.com/">Scott Logic</a> and is a prolific technical author, blogger and speaker on a range of technologies. He is a board member of <a hreflang="en" href="https://www.finos.org/">FINOS</a>, which is encouraging open source collaboration in the financial sector. He is also very active on GitHub, contributing to a number of different projects.
 results: https://docs.google.com/spreadsheets/d/11jyABro0fKtuN6INnYP9pJlv5QWwp0jfJyTsGfKgScg/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-unedited: true
+featured_quote: Despite being a niche technology, WebAssembly is already adding value to the web. There are a number of web applications that benefit greatly from this technology.
+featured_stat_1: 383
+featured_stat_label_1: Unique WebAssembly binarys discovered
+featured_stat_2: 34.9%
+featured_stat_label_2: Mobile sites using Amazon IVS
+featured_stat_3: 72.8%
+featured_stat_label_3: Modules using Emscripten
```

Comment:
I threw in a quote and some stats I picked myself. Feel free to change.

Reply:
Nits: 1) "binaries" not "binarys" I think; 2) not sure worth focusing on "amazon ivs", almost looks like sponsorship quote.

Reply:
Fix binaries in #3175 

Will let @ColinEberhardt to decide if they wanna replace the Amazon one. As I say I just picked a few to have some for launch. Personally I'm not worried about it looking like sponsorship. We list Google Fonts in Fonts chapter for example because they are the biggest font provider.